quetz/tests/api/test_channels.py

```diff
@@ -232,3 +232,42 @@ def test_unique_channel_names_are_case_insensitive(auth_client, user, db):
 
     assert response.status_code == 409
     assert f"{channel_name.upper()} exists" in response.json()['detail']
+
+
+def test_unicode_channel_names(auth_client, maintainer):
+
+    channel_name = "검은맘바"
+
+    response = auth_client.post(
+        "/api/channels", json={"name": channel_name, "private": False}
+    )
+
+    assert response.status_code == 201
+
+    response = auth_client.get("/api/channels")
+
+    assert response.status_code == 200
+    assert len(response.json()) == 1
+    assert response.json()[0]['name'] == channel_name
+
+    response = auth_client.get(f"/api/channels/{channel_name}")
+
+    assert response.status_code == 200
+    assert response.json()['name'] == channel_name
+
+
+def test_accents_make_unique_channel_names(auth_client, maintainer):
+
+    channel_names = ["żmija", "zmija", "grün", "grun"]
```

Comment:
does this mean we allow all 4 channel names to exist?

This also might allow for vicious attacks. Unicode has many characters that look similar.

For example: c⃝nda-f⃝rge

Reply:
ok this example doesn't work but you get the idea :)